.i18nrc.json

```
@@ -16,6 +16,7 @@
     "xpack.grokDebugger": "x-pack/plugins/grokdebugger",
     "xpack.idxMgmt": "x-pack/plugins/index_management",
     "xpack.licenseMgmt": "x-pack/plugins/license_management",
+    "xpack.machineLearning": "x-pack/plugins/ml",
```

Let's be consistently with #25466:
```suggestion
    "xpack.ml": "x-pack/plugins/ml",
```